src/clients/FtpGateway.cc

```diff
@@ -970,8 +970,10 @@ Ftp::Gateway::processReplyBody()
         return;
     }
 
+    const auto csize = data.readBuf->contentSize();
```

Comment:
Please use a more descriptive name. For example:

```suggestion
    const auto bufferedDataSize = data.readBuf->contentSize();
```

or

```suggestion
    const auto availableDataSize = data.readBuf->contentSize();
```

Reply:
Ok, let's rename it.